grpc-interface-support/README.md

```diff
@@ -24,7 +24,8 @@ For generating a client SDK:
         {
             "type": "grpc-interface",
             "config": {
-                "src": "<uri_to_proto_file>",
+                "src": "<uri_or_local_file_or_dir_path_to_proto_files>",
```

Comment:
Should we possibly in this file give some examples on src declarations that we support, as we likely do not support all possible URI schemese. Like have we tested if `file` URI works, like `file:///home/benjamin/my_file.txt`?

Reply:
Good idea, examples are always a good idea IMO. AFAIR we decided against `file://`.